src/cares_wrap.cc

```diff
@@ -1093,7 +1097,7 @@ static void IsIPv4(const FunctionCallbackInfo<Value>& args) {
   node::Utf8Value ip(args.GetIsolate(), args[0]);
   char address_buffer[sizeof(struct in_addr)];
 
-  if (uv_inet_pton(AF_INET, *ip, &address_buffer) == 0) {
+  if (args.Length() && uv_inet_pton(AF_INET, *ip, &address_buffer) == 0) {
```

Comment:
Shouldn't this type of extra check ideally be made *before* we start using `args` in this function?